lib/LedgerSMB/Report.pm

```diff
@@ -281,8 +281,8 @@ sub _render {
                        $srt_b = $srt_b->to_sort
                            if eval { $srt_b->can('to_sort') };
                    }
-                   no warnings 'numeric';
-                   $srt_a <=> $srt_b or $srt_a cmp $srt_b;
+
+                   $srt_a cmp $srt_b;
```

Comment:
There is no good algorithm for a universal sort of user data.  There is a design issue of not bringing type info in with the rows.

I will put it back as it was with a no-critic.  If this code is going to remain, I will rewrite the current sort.